extensions/amp-story/1.0/amp-story-system-layer.css

```diff
@@ -121,24 +121,24 @@
 }
 
 [info] .i-amphtml-message-container {
-  /* 48px for info button, add width to padding for 
-  * additional story buttons */
-  padding-right: 48px !important; 
+  /* 48px for info button, add width to padding for additional story buttons. */
+  padding-right: 48px !important;
 }
 
 [info][dir=rtl] .i-amphtml-message-container {
-  /* 48px for info button, add width to padding for 
-  * additional story buttons */
-  padding-right: auto !important; 
-  padding-left: 48px !important; 
+  /* 48px for info button, add width to padding for additional story buttons. */
+  padding-right: auto !important;
+  padding-left: 48px !important;
 }
 
 .i-amphtml-message-container {
   width: max-content !important;
   transition-property: opacity, transform !important;
   position: absolute !important;
-  top: 14px !important;
+  top: 12px !important;
   right: 48px !important;
+  color: #fff !important;
+  font-shadow: 0px 0px 6px rgba(0, 0, 0, 0.24) !important;
```

Comment:
Shouldn't this be `text-shadow`?

Reply:
I amended my commit and force pushed to pretend I never did that mistake :)))
Thanks for catching it!!